extensions/adobe/experience/decisioning/experienceevent-proposition-interaction.example.2.json

```diff
@@ -35,21 +22,8 @@
           }
         }
       ],
-      "xdm:decisionEventID": "3cc33a7e-13ca-4b19-b25d-c816eff9a70a",
-      "xdm:decisionEventDetails": {
-        "xdm:experienceID": "8A0FBA334DA23A64AD9D748E58533857B6B5556AB28328DCCC04DAE6E9F35430"
-      },
-      "xdm:decisionScope": "scope:12cfc3fa94281acb",
-      "xdm:decisionScopeDetails": {
-        "xdm:id": "scope:12cfc3fa94281acb",
-        "xdm:decisionProvider": "AEP Decisioning",
-        "xdm:placement": {
-          "xdm:id": "offer-placement:1175009612b0100c"
-        },
-        "xdm:activity": {
-          "xdm:id": "offer-activity:11cfb1fa93381aca"
-        }
-      }
```

Comment:
Remove the deprecated properties from the example and only leave the ones that got renamed